app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command):
+    file_to_copy = command.split()[1]
+    new_file = command.split()[2]
+    if file_to_copy == new_file:
+        return
+    with open(file_to_copy, "r") as file_one, open(new_file, "a") as file_two:
```

Comment:
```suggestion
    with open(file_to_copy, "r") as source, open(new_file, "a") as destination:
```
Use more descriptive names for file variables